definitions/divorce/json/CaseType.json

```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "DIVORCE",
-    "Name": "Divorce case - v114.22",
+    "Name": "Divorce case - v114.23",
```

Comment:
just a curiosity, do we need to bump up the minor version whenever we make changes to CCD?

Reply:
I was told that we should, but not everyone follows it.

Reply:
👍

Reply:
The idea is bump up the  version with each change to know the version of the config uploaded, but I think we need to find a better approach to avoid the conflicts